src/azure-cli/azure/cli/command_modules/acr/import.py

```diff
@@ -94,41 +96,44 @@ def acr_import(cmd,
                                               target_tags=target_tags,
                                               untagged_target_repositories=repository,
                                               mode=ImportMode.force.value if force else ImportMode.no_force.value)
-    result_poller = client.import_image(
-        resource_group_name=resource_group_name,
-        registry_name=registry_name,
-        parameters=import_parameters)
 
-    return _handle_result(cmd, result_poller, source_registry, source_image, registry)
+    try:
+        if no_wait:
+            return sdk_no_wait(no_wait, client.import_image, resource_group_name, registry_name, import_parameters)
+        else:
```

Comment:
@CarolineNB pylint is flagging this if/else with the "Unnecessary "else" after "return"" style error. You can see how to fix this by referencing [no-else-return](https://pheanex.github.io/pylint/#r1705-no-else-return).

Pylint error: https://dev.azure.com/azure-sdk/public/_build/results?buildId=973250&view=logs&jobId=36dd4138-4d53-5e46-00d9-e5cd9744be05&j=36dd4138-4d53-5e46-00d9-e5cd9744be05&t=1cf3879c-0a42-5ccd-7693-7a4781d739d8